bricksrc/quantities.py

```diff
@@ -1,109 +1,565 @@
+                    ?unit qudt:symbol ?symbol .
+                    FILTER(isLiteral(?symbol))
+                    }}"""
+    )
+    for r in res:
+        yield r
+
+
+"""
+Each is a qudt:QuantityKind
+"""
 quantity_definitions = {
     "Air_Quality": {
-        "subclasses": {
-            "CO2_Level": {},
-            "PM10_Level": {},
-            "PM25_Level": {},
-            "TVOC_Level": {},
+        "subconcepts": {
+            "CO2_Level": {
+                QUDT.applicableUnit: [UNIT.PPM],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                SKOS.definition: Literal("The concentration ratio of some substance"),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("Concentration"),
+                SKOS.broader: QUDTQK.Dimensionless,
+            },
+            "PM10_Level": {
+                QUDT.applicableUnit: [UNIT.PPM],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                SKOS.definition: Literal("The concentration ratio of some substance"),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("Concentration"),
+                SKOS.broader: QUDTQK.Dimensionless,
+            },
+            "PM25_Level": {
+                QUDT.applicableUnit: [UNIT.PPM],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                SKOS.definition: Literal("The concentration ratio of some substance"),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("Concentration"),
+                SKOS.broader: QUDTQK.Dimensionless,
+            },
+            "TVOC_Level": {
+                QUDT.applicableUnit: [UNIT.PPM],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                SKOS.definition: Literal("The concentration ratio of some substance"),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("Concentration"),
+                SKOS.broader: QUDTQK.Dimensionless,
+            },
         },
     },
-    "Angle": {},
-    "Conductivity": {},
-    "Capacity": {},
+    "Angle": {OWL.sameAs: QUDTQK["Angle"]},
+    "Conductivity": {OWL.sameAs: QUDTQK["Conductivity"]},
+    "Capacity": {OWL.sameAs: QUDTQK["Capacity"]},
     "Enthalpy": {
         SKOS.definition: Literal(
             "(also known as heat content), thermodynamic quantity equal to the sum of the internal energy of a system plus the product of the pressure volume work done on the system. H = E + pv, where H = enthalpy or total heat content, E = internal energy of the system, p = pressure, and v = volume. (Compare to [[specific enthalpy]].)"
         ),
+        OWL.sameAs: QUDTQK["Enthalpy"],
+    },
+    "Mass": {
+        OWL.sameAs: QUDTQK["Mass"],
+        "subconcepts": {
+            "Grains": {
```

Comment:
definition missing.

Reply:
When there is a `OWL.sameAs` property, the definition is provided by QUDT. Is that what you're referring to?

Reply:
I was referring to `Grains`. I don't see it is owl.sameAs something in QUDT.

Reply:
It wasn't clear what you were referring to. In the future, please make sure you're explicit about exactly which term/aspect you're talking about so that we don't have to have this unnecessary back and forth